packages/apps/plugins/plugin-script/src/components/FrameContainer/frame.tsx

```diff
@@ -38,9 +38,9 @@ const services = new ClientServicesProxy(
 
 // TODO(burdon): Error handling.
 createRoot(document.getElementById('root')!).render(
-  <ClientProvider services={() => services}>
-    <div className='flex fixed inset-0'>
+  <div className='flex fixed inset-0'>
```

Comment:
Please avoid using existing fragments for this — `fixedInsetFlexLayout` applies these styles (and some more). If you need a different fragment, we should discuss.

Generally we should avoid proliferating `fixed inset-0` elements, this approach has introduced significant tech debt before. What problem to those styles solve in this case?

Reply:
I see this isn’t actually new for this plugin, so my comment isn’t blocking, just food for thought for now.